Mixpanel/MPVariant.m

```diff
@@ -102,24 +102,28 @@ + (MPVariant *)variantWithJSONObject:(NSDictionary *)object {
         return nil;
     }
 
+    BOOL targeting = [[object objectForKey:@"targeting"] boolValue];
```

Comment:
What if this key doesn't exist. As a rule we are pretty strict about validation in these initializer class methods, so that we don't have to do any validation in the constructor.